libs/components/tiles/testing/src/modules/tiles/tile-harness.ts

```diff
@@ -0,0 +1,129 @@
+import { HarnessPredicate } from '@angular/cdk/testing';
+import { SkyComponentHarness } from '@skyux/core/testing';
+import { SkyHelpInlineHarness } from '@skyux/help-inline/testing';
+import { SkyChevronHarness } from '@skyux/indicators/testing';
+
+import { SkyTileContentHarness } from './tile-content-harness';
+import { SkyTileHarnessFilters } from './tile-harness-filters';
+
+/**
+ * Harness to interact with a tile component in tests.
+ */
+export class SkyTileHarness extends SkyComponentHarness {
+  /**
+   * @internal
+   */
+  public static hostSelector = 'sky-tile';
+
+  #getChevron = this.locatorFor(SkyChevronHarness);
+  #getContent = this.locatorFor(SkyTileContentHarness);
+  #getTitle = this.locatorFor('sky-tile-title');
+  #getSummary = this.locatorFor('sky-tile-summary');
+  #getSettings = this.locatorForOptional('button.sky-tile-settings');
+
+  /**
+   * Gets a `HarnessPredicate` that can be used to search for a
+   * `SkyTileHarness` that meets certain criteria.
+   */
+  public static with(
+    filters: SkyTileHarnessFilters,
+  ): HarnessPredicate<SkyTileHarness> {
+    return SkyTileHarness.getDataSkyIdPredicate(filters);
+  }
```

Comment:
Can you add a filter for consumers to get a tile from its title?

Reply:
added